web/router.ex

```diff
@@ -25,8 +25,8 @@ defmodule Cog.Router do
     resources "/v1/users/reset-password", V1.PasswordResetController, only: [:create, :update]
 
     resources "/v1/groups", V1.GroupController
-    get "/v1/groups/:id/users", V1.GroupMembershipController, :index
-    post "/v1/groups/:id/users", V1.GroupMembershipController, :manage_group_users
+    get "/v1/groups/:id/users", V1.GroupController, :show
+    post "/v1/groups/:id/users", V1.GroupController, :manage_group_users
```

Comment:
They shouldn't. The index function in the membership controller was an exact duplicate of the show function in the group controller and the manage_group_users function is just routing the logic through the update function. The api should remain unchanged.

My thinking with routing the group users bits through the update function was that we could eventually drop the extra routes with a future update. But for now everything should function the same as before.